cnstrc_README.md

```diff
@@ -36,4 +36,26 @@ kenlm_bin.call(
     'build_binary', ['-s', '<file_in>', '<file_out>'],
     timeout=60,
 )
-```
\ No newline at end of file
+```
+
+## manylinux wheels
+To build cross platform **manylinux** wheels run the following from project root:
+```console
+docker build -t manylinux2014_for_kenlm .
+docker run --rm -v ${PWD}:/io -w=/io manylinux2014_for_kenlm /bin/bash build_cnstrc_manylinux_wheel.sh
+```
+
+This will mount your working directory to Docker container and use it to build wheels and manylinux
+wheels. The results of the build will be in `./wheels/manylinux`.
+
+## macOS wheels
+Requirements:
+ - be on macOS
+ - have [delocate](https://github.com/matthew-brett/delocate) installed
```

Comment:
maybe better we also add a command to install here with a minimum version that worked? 
```
pip install delocate>=0.10.1
```

Reply:
https://github.com/Constructor-io/kenlm/pull/2/commits/4e33d2919890243d28cae35e1b397777d7c7b23f